db/data_migrations/20180920104841_populate_teams_for_overturned_fois.rb

```diff
@@ -0,0 +1,16 @@
+class PopulateTeamsForOverturnedFois < ActiveRecord::DataMigration
+  def up
+    overturned_foi_ct = CorrespondenceType.overturned_foi
+    foi_ct = CorrespondenceType.foi
+
+    teams = BusinessUnit.all
+    teams.each do |bu|
+      if bu.correspondence_types.include?(foi_ct)
+        unless bu.correspondence_types.include?(overturned_foi_ct)
+          bu.correspondence_types << overturned_foi_ct
+          bu.save!
+        end
+      end
```

Comment:
I understood that what we were going to do is re-use the FOI correspondence type so that if a BU can handle FOIs, they can handle Overturned FOIs too.